Src/Fare/Fare.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\Common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net35;netstandard1.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
I might have to switch this back to how it was if you don't mind. Upgrading silently to .NET Standard 2.0 can break existing clients that are on .NET Standard 1.1.

Reply:
That's fine; I wasn't able to test below that level at the time.